PubnativeLite/PubnativeLite/HyBidSettings.m

```diff
@@ -72,6 +72,21 @@ - (NSString *)deviceName {
     return currentDevice.model;
 }
 
+- (CGSize) getOrientationIndependentScreenSize {
+    return CGSizeMake(MIN([UIScreen mainScreen].bounds.size.width,
+                          [UIScreen mainScreen].bounds.size.height),
+                      MAX([UIScreen mainScreen].bounds.size.width,
+                          [UIScreen mainScreen].bounds.size.height));
+}
```

Comment:
Is there a reason why you use MIN and MAX here?

Reply:
Yeah @faresbh . As I said in the description starting from iOS8 screen size & width are orientation dependent. To get a size that is independent from orientation, we need to compare the values and choose the right one. (The minimum is for the width and the maximum is for the height) Otherwise width and height values will change based on device's current orientation.